simphony_ui/openfoam_model.py

```diff
@@ -29,15 +30,15 @@ class OpenfoamModel(HasStrictTraits):
     #: The number of iterations in the simulation.
     num_iterations = PositiveInt(10)
 
+    #: The boundary conditions during the simulation
+    boundary_conditions = Instance(BoundaryConditionsModel)
+
     #: The viscosity of the fluid.
     viscosity = PositiveFloat(1.0e-3)
 
     #: The density of the fluid.
     density = PositiveFloat(1000.0)
 
-    #: The pression difference.
```

Comment:
I'm not sure, if you look at the main script you can see that delta_p is only used in the boundary conditions